examples/multicluster_heal/README.md

```diff
@@ -0,0 +1,101 @@
+# Basic floating interdomain examples
```

Comment:
Do we need to duplicate a multicluster setup?
Can we use the same approach as for basic/heal setup?

Reply:
@NikitaSkrynnik Could you have a look?